src/index.jsx

```diff
@@ -49,27 +55,62 @@ function Form() {
         setPref('app', app);
     }, [app])
 
+    const handleSelectChange = (event) => {
+        console.log(event.target)
+        const selectedItemKey = event.target.value;
+        setApp(selectedItemKey);
+    };
+
+    const options = Object.entries(AVAILABLE_APPS).map(([key, value]) => (
+        <option value={key} key={key}>{value.title}</option>
+    ));
+
     return <Box display="flex" flexDirection="column">
         <Heading sx={{ fontSize: 2, mb: 1 }}>JupyterHub URL</Heading>
 
-        <TextInput value={hubUrl} onChange={(ev) => setHubUrl(ev.target.value)} placeholder="https://myjupyterhub.org" aria-label="JupyterHub URL" />
+        <TextInput
+            value={hubUrl}
+            onChange = {
+                (ev) => setHubUrl(ev.target.value)
+            }
+            placeholder="https://myjupyterhub.org"
+            aria-label="JupyterHub URL"
+            sx={{ pt: 0.5, pb: 0.5 }}
+        />
+
```

Comment:
spacing changes. This is the intended use of sx, and not exactly the same as inline CSS.